app/views/recipes/public.html.erb

```diff
@@ -0,0 +1,22 @@
+<h1 class="d-flex justify-content-center">Public recipes</h1>
+<div>
+  <% @public_recipes.each do |recipe| %>
+    <div class="card mb-3">
+      <div class="card-body">
```

Comment:
- [ ] Clicking on a recipe item in Public recipes page should lead to recipe details. But in your application I, on public page, cannot click on the recipe .